networkx/algorithms/independence.py

```diff
@@ -0,0 +1,52 @@
+import networkx as nx
+from networkx.algorithms import *
```

Comment:
using `from ... import *` is a bad practice. Some client IDEs may expose the unused functions imported by this statement, cluttering up the namespace of the client.